dashboard/app/models/framework.rb

```diff
@@ -14,4 +14,12 @@
 #  index_frameworks_on_shortcode  (shortcode) UNIQUE
 #
 class Framework < ApplicationRecord
```

Comment:
Does this need a has_many association to categories?

Reply:
Not yet. I will add this when needed.